covidWallet/ios/Podfile

```diff
@@ -74,10 +72,13 @@ target 'ZADA' do
       config[:reactNativePath],
       :mac_catalyst_enabled => false
     )
+    installer.pods_project.build_configurations.each do |config|
+      config.build_settings["EXCLUDED_ARCHS[sdk=iphonesimulator*]"] = "arm64"
+    end
```

Comment:
Why is this written again? It was already included. Please see lines 79 - 81.

Reply:
Both of them are different.

- The first block applies the exclusion setting globally to the entire project’s build configurations.
- The second block applies the exclusion setting to each target’s build configurations individually.